velox/dwio/common/DirectBufferedInput.cpp

```diff
@@ -274,6 +274,7 @@ std::vector<cache::CachePin> DirectCoalescedLoad::loadData(bool isPrefetch) {
   }
   input_->read(buffers, requests_[0].region.offset, LogType::FILE);
   ioStats_->read().increment(size);
+  ioStats_->incRawBytesRead(size - overread);
```

Comment:
@JkSelf Can you also fix this in `DwioCoalescedLoadBase::updateStats`?  Thanks

Reply:
@Yuhta Updated. Please help to review again. Thanks.

Reply:
@Yuhta The rawBytesRead  already updated in `loadSync `and `loadData`. It seems no need updated in `DwioCoalescedLoadBase::updateStats`.

Reply:
I have reviewed the code and found that the update of `rawBytesRead `for `CachedInputBuffer `is done in the `loadSync `function. `loadSync `is executed unconditionally after `loadOrFuture `[here](https://github.com/facebookincubator/velox/blob/main/velox/dwio/common/CacheInputStream.cpp#L317). However, in the case of `DirectInputBuffer`, the execution of `loadSync `is conditional [here](https://github.com/facebookincubator/velox/blob/main/velox/dwio/common/DirectInputStream.cpp#L193), so we need to update `rawBytesRead `in the `loadData `function instead.

Reply:
@Yuhta Do you have any further comment? Thanks.

Reply:
Thanks for the explanation.  Can you add comment to those 2 places saying the counter is increased in `loadSync` unconditionally so we don't need to do it in `loadData`?

Reply:
@Yuhta Added the comments. Please help to review again. Thanks.